### DISCUSSION TITLE
School management app

### DISCUSSION THREAD

**Discussion by Provkr71** [ketanchoyal/Our-E-School#33] [General]
Can we break this one app into three app (Super admin app, Teacher app, and student app)

**Comment by kzawadi** [comment]
You have to have a very compelling reason for such a break-up as it introduces 3 separate codebases* which are a kinder bad ways to go. Also unfortunately this repository has no timeline for when it will be migrated to null safety code. But the architecture is so good so far things like introducing new features is as smooth as butter.
